e2e/interchaintestv8/relayer_test.go

```diff
@@ -233,3 +269,216 @@ func (s *RelayerTestSuite) TestRelayToEth() {
+				SourceChannel:      sendPacket.SourceChannel,
+				DestinationChannel: sendPacket.DestChannel,
+				TimeoutTimestamp:   sendPacket.TimeoutTimestamp,
+				Payloads: []channeltypesv2.Payload{
+					{
+						SourcePort:      sendPacket.Payloads[0].SourcePort,
+						DestinationPort: sendPacket.Payloads[0].DestPort,
+						Version:         sendPacket.Payloads[0].Version,
+						Encoding:        sendPacket.Payloads[0].Encoding,
+						Value:           sendPacket.Payloads[0].Value,
+					},
+				},
+			}
+			recvPacketMsgs[i] = &channeltypesv2.MsgRecvPacket{
+				Packet:          packet,
+				ProofCommitment: storageProofBz,
+				ProofHeight: clienttypes.Height{
+					RevisionNumber: 0,
+					RevisionHeight: s.LastEtheruemLightClientUpdate,
+				},
+				Signer: cosmosUserAddress,
+			}
+		}
+
+		resp, err := s.BroadcastMessages(ctx, simd, cosmosUserWallet, 20_000_000, recvPacketMsgs...)
+		s.Require().NoError(err)
+		s.Require().NotEmpty(resp.TxHash)
+
+		txHash, err = hex.DecodeString(resp.TxHash)
+		s.Require().NoError(err)
+
+		s.Require().True(s.Run("Verify balances on Cosmos chain", func() {
+			denomOnCosmos := transfertypes.NewDenom(s.contractAddresses.Erc20, transfertypes.NewHop(transfertypes.PortID, ibctesting.FirstChannelID))
+
+			// User balance on Cosmos chain
+			resp, err := e2esuite.GRPCQuery[banktypes.QueryBalanceResponse](ctx, simd, &banktypes.QueryBalanceRequest{
+				Address: cosmosUserAddress,
+				Denom:   denomOnCosmos.IBCDenom(),
+			})
+			s.Require().NoError(err)
+			s.Require().NotNil(resp.Balance)
+			s.Require().Equal(totalTransferAmount.Uint64(), resp.Balance.Amount.Uint64())
+			s.Require().Equal(denomOnCosmos.IBCDenom(), resp.Balance.Denom)
+		}))
+	}))
+
+	s.Require().True(s.Run("Acknowledge packets on Ethereum", func() {
+		var multicallTx []byte
+		s.Require().True(s.Run("Retrieve relay tx to Ethereum", func() {
+			resp, err := s.RelayerClient.RelayByTx(context.Background(), &relayertypes.RelayByTxRequest{
+				SourceTxIds:     [][]byte{txHash},
+				TargetChannelId: s.TendermintLightClientID,
+			})
+			s.Require().NoError(err)
+			s.Require().NotEmpty(resp.Tx)
+			s.Require().Equal(resp.Address, ics26Address.String())
+
+			multicallTx = resp.Tx
+		}))
+
+		s.Require().True(s.Run("Submit relay tx to Ethereum", func() {
+			ethClient, err := ethclient.Dial(eth.RPC)
+			s.Require().NoError(err)
+
+			txOpts := s.GetTransactOpts(s.key, eth)
+			s.Require().NoError(err)
+
+			tx := ethtypes.NewTransaction(
+				txOpts.Nonce.Uint64(),
+				ics26Address,
+				txOpts.Value,
+				5_000_000,
+				txOpts.GasPrice,
+				multicallTx,
+			)
+
+			signedTx, err := txOpts.Signer(txOpts.From, tx)
+			s.Require().NoError(err)
+
+			// Submit the relay tx to Ethereum
+			err = ethClient.SendTransaction(ctx, signedTx)
+			s.Require().NoError(err)
+
+			// Wait for the tx to be mined
+			receipt := s.GetTxReciept(ctx, eth, signedTx.Hash())
+			s.Require().Equal(ethtypes.ReceiptStatusSuccessful, receipt.Status)
```

Comment:
Since we are testing the relayer, would it also make sense to check if the acks went through as expected? Either by checking that commitments have been deleted or checking the events from the router?